libs/common-services/src/main/java/uk/gov/di/ipv/core/library/config/CoreFeatureFlag.java

```diff
@@ -3,7 +3,8 @@
 public enum CoreFeatureFlag implements FeatureFlag {
     UNUSED_PLACEHOLDER("unusedPlaceHolder"),
     RESET_IDENTITY("resetIdentity"),
-    INHERITED_IDENTITY("inheritedIdentity");
+    INHERITED_IDENTITY("inheritedIdentity"),
+    REPROVE_IDENTITY_AUDIT_ENABLED("reproveIdentityAuditEnabled");
```

Comment:
Maybe rename `reproveIdentityEnabled` as this isn't just about the audit event, it's the reprove identity feature as a whole